citation-react-renderer/src/urls.js

```diff
@@ -2,8 +2,9 @@ export default function getPaths(pages, parent = { url: '', pages: [] }) {
 	return pages.reduce((acc, page) => {
 		const children = page.children ? page.children : [];
 		let url;
-		if (parent.url === '' && (page.slug === '' || page.slug === '/')) {
+		if (parent.url === '' && (!page.slug || page.slug === '' || page.slug === '/')) {
 			url = '';
+			page.slug = '';
```

Comment:
Not really satisfied with this solution because I don't like the modifying input parameters but didn't find a better way. If someone has a better idea, I'll be glad to change it